chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -2880,7 +2811,9 @@ public synchronized void importFromJson(JsonObject json) {
     yMax = json.get("yMax").asInt(Math.min(yClipMax, yMax));
 
     exposure = json.get("exposure").doubleValue(exposure);
-    postprocess = Postprocess.get(json.get("postprocess").stringValue(postprocess.name()));
+    postProcessingFilter = PostProcessingFilters
+            .getPostProcessingFilterFromId(json.get("postprocess").stringValue(postProcessingFilter.getId()))
+            .orElse(PostProcessingFilters.getDefault());
```

Comment:
Maybe add a warning, something like this:

```suggestion
            .orElse(() -> {
              if (json.get("postprocess").stringValue(null) != null) {
                Log.warn("The post processing filter " + json + " is unknown. Maybe you're missing a plugin that was used to create this scene?");
              }
              return PostProcessingFilters.getDefault();
            });
```